lms/templates/ccx/schedule.html

```diff
@@ -29,80 +29,88 @@
+        <button id="save-changes" aria-describedby="message_save" tabindex="2">${_("Save changes")}</button>
       </div>
     </form>
   </div>
-  <div class="ccx-sidebar-panel" id="ajax-error">
+  <div class="ccx-sidebar-panel" id="ajax-error" tabindex="1" role="group" aria-labelledby="ccx_schedule_error_message">
     <h2>${_('Error')}</h2>
-    <p>${_("There was an error saving changes.")}</p>
+    <p id="ccx_schedule_error_message">${_("There was an error saving changes.")}</p>
   </div>
-  <div class="ccx-sidebar-panel">
-    <h2>${_('Schedule a Unit')}</h2>
+  <div class="ccx-sidebar-panel" tabindex="2" aria-labelledby="ccx_schedule_unit"
+       id="ccx_schedule_unit_panel" role="group">
+    <h2 id="ccx_schedule_unit">${_('Schedule a Unit')}</h2>
     <form role="form" id="add-unit" name="add-unit" class="ccx-form">
       <div class="field">
-        <b>${_('Section')}</b><br/>
-        <select name="chapter"></select>
+        <label for="ccx_chapter"><b>${_('Section')}</b></label>
+        <select name="chapter" id="ccx_chapter" tabindex="2" ></select>
       </div>
       <div class="field">
-        <b>${_('Subsection')}</b><br/>
-        <select name="sequential"></select>
+        <label for="ccx_sequential"><b>${_('Subsection')}</b></label>
+        <select name="sequential" id="ccx_sequential" tabindex="2"></select>
       </div>
       <div class="field">
-        <b>${_('Unit')}</b><br/>
-        <select name="vertical"></select>
+        <label for="ccx_vertical"><b>${_('Unit')}</b></label>
+        <select name="vertical" id="ccx_vertical" tabindex="2"></select>
       </div>
       <div class="field datepair">
-        <b>${_('Start Date')}</b><br/>
-        <input placeholder="Date" type="date" class="date" name="start_date"/>
-        <input placeholder="time" type="time" class="time" name="start_time"/>
+        <label for="ccx_start_date"><b>${_('Start Date')}</b></label>
+        <input placeholder="yyyy-mm-dd" type="text" class="date" name="start_date" id="ccx_start_date"
+               aria-label="${_('Start Date')} format yyyy-mm-dd" tabindex="2"/>
+        <label for="ccx_start_time" class="sr">${_('Start Time format HH:mm')}</label>
+        <input placeholder="time" type="text" class="time" name="start_time" id="ccx_start_time" tabindex="2"/>
       </div>
       <div class="field datepair">
-        <b>${_('Due Date')}</b> ${_('(Optional)')}<br/>
-        <input placeholder="Date" type="date" class="date" name="due_date"/>
-        <input placeholder="time" type="time" class="time" name="due_time"/>
+        <label for="ccx_due_date"><b>${_('Due Date')}</b> ${_('(Optional)')}</label>
+        <input placeholder="yyyy-mm-dd" type="text" class="date" name="due_date" id="ccx_due_date"
+                aria-label="${_('Due Date Optional')} format yyyy-mm-dd" tabindex="2"/>
+        <label for="ccx_due_time" class="sr">${_('Due Time format HH:mm')}</label>
```

Comment:
I like that you added screen reader text here to help with formatting.  However, screen readers don't read HH:mm very well as its not a word.  In fact, it was unintelligible.  I would recommend spelling it out "hour, hour, colon, minute, minute"